pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/TlsIntegrationTest.java

```diff
@@ -524,6 +524,9 @@ public void testJDBCClient()
     ResultSet resultSet = statement.executeQuery(query);
     resultSet.first();
     Assert.assertTrue(resultSet.getLong(1) > 0);
+    resultSet.close();
```

Comment:
Consider using try-with-resource?
We should also close them for the following invalid connection